python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -159,7 +159,7 @@ def _set_login_message(self, message):
         Set the message in the dialog.
         :param message: Message to display in the dialog.
         """
-        self.ui.message.setText(message)
+        self.ui.message.setText(str(message))
```

Comment:
I'm unclear why you need to cast message to a str here - was there a specific problem that this solves?  Might be worth mentioning this in the param description or (possibly preferably) fixing it at source wherever _set_login_message is called from?